app/src/Home/Home.js

```diff
@@ -1,20 +1,221 @@
+                    <label htmlFor="email" className="text-black text-[3vmin]">My email is:</label>
+                    <input id="email" name="email" type="text" placeholder="..." maxLength={128} rows={1}
+                           className="w-[90%] border-[2px] border-enquirygray p-[1vmin] rounded-lg text-black text-[3vmin]
+                           focus:border-logodarkyellow focus:outline-none focus:ring-0"
+                           value={email}
+                           onChange={(e) => setEmail(e.target.value)} />
+                    <label htmlFor="message" className="text-black text-[3vmin]">Let's talk about:</label>
+                    <textarea id="message" name="message" placeholder="..." maxLength={512} rows={5}
+                           className="w-[90%] h-auto border-[2px] border-enquirygray p-[1vmin] rounded-lg text-black text-[3vmin]
+                           focus:border-logodarkyellow focus:outline-none focus:ring-0 text-start resize-none"
+                           value={message}
+                           onChange={(e) => setMessage(e.target.value)}></textarea>
+                    <input className="border-[2px] border-enquirygray rounded-md px-[2vmin] py-[1vmin] cursor-pointer duration-200 text-black text-[3vmin]
+                                     hover:border-logodarkyellow hover:bg-logodarkyellow hover:text-white" id="submit" name="submit" type="submit" value="Submit"></input>
+                </form>
+            </div>
+            <div className="absolute bottom-0 right-[4vmin] w-[45%] h-full flex flex-col justify-center items-center gap-[1vh] border-transparent border-y-[4vmin]">
+                <img className="absolute top-0 left-0 w-full h-full brightness-75" src="/resources/240_F_723820135_tcfTiQlaVjadRhGdI5XlLdvOmUeZmwmY.jpg" alt="Background" />
+                <p className="text-[8vmin] text-white font-serif font-bold z-10">Contact Us</p>
+                <img className="w-auto h-[50%] z-10" src="/resources/WHITE_LOGO_WITHOUT_TAGLINE[1].png" alt='Company Logo' />
+                <p className="text-[3vmin] text-white z-10">Mobile: +611 2345 67 89</p>
+                <p className="text-[3vmin] text-white z-10">Email: vinit@telestech.com</p>
+                <div className="w-auto h-[5vmin] grid grid-cols-5 gap-x-[2vmin] z-10 p-[1vmin]
+                                *:text-[4vmin] *:text-white">
+                    <a href="/">
+                        <SiInstagram />
+                    </a>
+                    <a href="/">
+                        <SiGithub />
+                    </a>
+                    <a href="/">
+                        <SiX />
+                    </a>
+                    <a href="/">
+                        <SiLinkedin />
+                    </a>
+                    <a href="/">
+                        <SiOnlyfans />
+                    </a>
+                </div>
+            </div>
+        </div>
+    );
```

Comment:
Having all the definitions in the same file will make it hard to maintain.
Move it to either:

```
Home
 |-> Components
    |-> <component>.js 
```

Or just create various `<component>.js` files under the `Home` directory.